sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -240,6 +241,10 @@ public void onOrientationChanged(int orientation) {
 
         // Flush any on-disk errors
         errorStore.flushOnLaunch();
+
+        if (config.getDetectNdkCrashes()) {
```

Comment:
Rather than making this conditional, the native interface should always be configured since it will be required by ANR detection.

`NativeBridge.install()` should instead receive `config.getDetectNdkCrashes()` as the `autoNotify` argument.

Reply:
_Insert in-person conversation_

This can stay as-is provided ANR detection is also added as a condition.

Reply:
If `detectNdkCrashes` and `detectAnrs` are both disabled, we can avoid initialising the NDK layer entirely. This is because it costs around 100ms to initialise the NDK layer and we should skip this if a user doesn't want any of that information. A potential risk is that this might introduce some new state to the notifier if users somehow trigger install of the NDK directly.